test/index/validator/server.test.ts

```diff
@@ -22,26 +22,23 @@
  *  OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
  *  OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
  *  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
- **/
+ */
 
-'use strict';
+import chai from 'chai';
+import _ from 'lodash';
 
-const
-	_ = require('lodash'),
-	chai = require('chai'),
+import { ServerValidator } from '../../../src/index/validator/server';
 
-	expect = chai.expect,
+const expect = chai.expect;
 
-	ServerValidator = require('../../../lib/index/validator/server');
-
-describe('index/validator/server.js', () => {
+describe('index/validator/server.ts', () => {
 
 	describe('constructor', () => {
 
 		it('should return the schema if it is valid', () => {
 
 			// Given
-			const config = {
+			const options: any = {
```

Comment:
COMMENT: Please avoid inflating what is already a big diff by renaming variables without legitimate reason (if there is a good reason, I digress).